### ISSUE TITLE
[Accessibility] "WinUI" marker icons are incorrectly tagged "presentation" and need alt text 

### ISSUE THREAD

**Issue by citelao** [MicrosoftDocs/windows-uwp#2791]
The [Button](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/buttons) overview page on MSDN (and perhaps other pages, though I could not find any in basic controls) has a table of standard buttons.

Some of these controls include a small WinUI icon, presumably to indicate that they are only present in WinUI:

![DropDownButton control description with a small WinUI icon](https://user-images.githubusercontent.com/432939/97913865-f6b0be00-1d1c-11eb-8fc4-33d70aa46c67.png)

* I hovered over the icon to see if there was alt text or a tooltip, but none appeared. 
* I opened Narrator to read the line, but Narrator read nothing about the icon.
* In the page source, I found no alt text and `role="presentation"`.

I think that is an accessibility bug. The icons are semantically meaningful (they are icons, not decorative images) and so they deserve alt text.

I am not sure what the icon means in the first place, and I'm not sure if the same issue occurs elsewhere, so I did not add the alt text myself 😅.

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0b6e57c0-4d39-7608-e535-f3bf35d3f7a2
* Version Independent ID: 826f8e2b-b5be-3c62-6bb6-85abea1bfba4
* Content: [Buttons - UWP applications](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/buttons#feedback)
* Content Source: [windows-apps-src/design/controls-and-patterns/buttons.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/controls-and-patterns/buttons.md)
* Product: **uwp**
* Technology: **design**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by hkhmsft** [comment]
Hi @citelao. Images of type "icon" don't actually support alt text. The text directly below helps explain those three controls:

> "DropDownButton, SplitButton, and ToggleSplitButton are included as part of the Windows UI Library, a NuGet package that contains new controls and UI features for Windows apps. For more info, including installation instructions, see Windows UI Library."

Hope this helps! Marking this issue as Closed but feel free to follow up in this thread.

**Comment by citelao** [comment]
I was actually raising a "meta" issue about the docs themselves, not about a bug in the XAML framework :D.

The Button page linked above includes a list of different types of buttons in XAML, and includes a little WinUI icon if (presumably) the subtype is only available via WinUI. The HTML for this page looks like this:

```html
<td>
  <img src="images/winui-logo-16x16.png" role="presentation" data-linktype="relative-path">
   A button with a chevron to open an attached flyout.
</td>
```

The image is marked `role="presentation"`, which tells screen readers that this image is not meaningful and can be skipped. This is incorrect, since the image is a visual indicator that a type of button is only available via WinUI. 

Instead, the `img` should look something like this:

```
<img src="images/winui-logo-16x16.png" data-linktype="relative-path" title="Only available in WinUI">
```

`alt` is technically more appropriate (describe the image), but I was confused what the icon meant *visually*—I only discovered this issue when I went searching for a tooltip. Using the `title` attribute as above will add a tooltip for confused folks like me.

---

I filed the bug rather than fixing myself for two reasons:

1. I'm not sure exactly what the button means.
2. I'm not sure where else this is a problem.

Thanks!
